assets/css/style.css

```diff
@@ -1,276 +1,398 @@
-.header-container {
+body {
+  background-color: var(--white);
+  width: 100%;
+  font-family: var(--font-family-style);
+}
+
+.top-mob {
+  height: 718px;
+  background-image: url('/assets/background-desk.png');
+  background-color: rgb(252, 246, 246);
```

Comment:
- [ ] Kindly add the correct path for the background image here to make it work online for this situation you can make it like that: `background-image: url('../background-desk.png');`
- [ ] Please remove the background color or make it white to match the Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/167130347-01aaeaa9-2330-4d4b-b336-3b517177368a.png) | ![image](https://user-images.githubusercontent.com/68796232/167130321-93f908e3-f83d-4584-a8ce-a8338882f43b.png)  |